packages/hooks/src/data/QueryData.ts

```diff
@@ -330,7 +332,8 @@ export class QueryData<TData, TVariables, TLazy> extends OperationData {
         ...result,
         data: undefined,
         error: undefined,
-        loading: false
+        loading: false,
+        called: true
```

Comment:
I wonder if it shouldn't be `false` as well when skipping. Semantically, the query wasn't called, but it might have been called before and the `skip` was set later. That would mean it should derive the value from the previous result, but since it's `true` by default as well, it doesn't matter.

So I guess it's easier to document that non-lazy queries have a `called` always true.